src/buildercore/cfngen.py

```diff
@@ -40,7 +40,10 @@ def build_context(pname, **more_context): # pylint: disable=too-many-locals
         del more_context['existing_context']
 
     def from_existing_context(field, default_value):
-        return existing_context.get(field, default_value)
+        password = existing_context.get(field)
+        if password:
+            return password
+        return default_value
```

Comment:
Is this because of empty strings/`None` values?

Reply:
Just seen commit.

Reply:
After https://github.com/elifesciences/builder/pull/191 we stopped regenerating the RDS super user password at every update, reusing the previously existing one if it's there. However, the "previously existing one" is `None` if there was no RDS in that stack; in that case, we need to actually generate it.